ftw/htmlblock/browser/resources/styles.scss

```diff
@@ -1 +1,15 @@
 @include portal-type-font-awesome-icon(ftw-htmlblock-htmlblock, code);
+
+.sl-can-edit .ftw-htmlblock-htmlblock {
```

Comment:
This will probably not work with the editmode toggle, since `sl-can-edit` is only based on the permission.